tests/test_event_builder.py

```diff
@@ -174,8 +255,7 @@ def test_create_conversion_event__with_attributes(self):
     }
 
     with mock.patch('time.time', return_value=42.123), \
```

Comment:
Why was this test changed?

Reply:
We don't need to mock bucketer when testing these EventBuilder methods.

Reply:
We do need to as it confirms that the test will show the right variation.

Reply:
Please restore the mock and then this will be good to go.